src/test/components/TrackMemory.test.js

```diff
@@ -142,6 +144,14 @@ describe('TrackMemory', function () {
     expect(getTooltipContents()).toMatchSnapshot();
   });
 
+  it('has a tooltip that matches the snapshot for counts equalling zero', function () {
+    const { moveMouseAtCounter, getTooltipContents } = setup({
+      countNumber: 0,
```

Comment:
Probably you want `-1` here instead, to make it closer to your use case.

Reply:
no, it works when <= 0 is excluded, I'll change it to the new "not present" value